app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    copy_data = command.split()
+
+    if len(copy_data) < 3 or copy_data[0] != "cp":
+        return
+
+    command, file_to_copy_name, new_file_name = copy_data
```

Comment:
Unpacking `copy_data` into `command, file_to_copy_name, new_file_name` assumes that `copy_data` always has exactly three elements. This can lead to a `ValueError` if the input does not meet this condition. Consider checking the length of `copy_data` before unpacking to avoid potential errors.